roles/contiv_network/tasks/aci_tasks.yml

```diff
@@ -23,6 +23,7 @@
   service: name=aci-gw state=started
   run_once: true
 
+# NOTE: Use Contivctl command when it is available
 - name: set aci mode
-  shell: contivctl global set -nwinfra aci
```

Comment:
@DivyaVavili, @erikh 

I tried digging in a bit. The current contivctl release is a bit old and doesn't include the latest netctl command additions. I tried publishing a new release but it failed (http://contiv.ngrok.io/view/Misc/job/Contivctl_Release/11/console).

The new `version` related changes in netplugin seem to depend on generated code which seem to be the root-cause of this failure. I will see if there is a way to generate that code in contivctl without taking away it's simple approach.

Please share if you have some ideas on handling this.

ping @jainvipin @shaleman .